subsys/net/l2/ieee802154/ieee802154.c

```diff
@@ -190,6 +253,12 @@ static enum net_verdict ieee802154_recv(struct net_if *iface,
 		return NET_DROP;
 	}
 
+	/* validate LL destination address (when IEEE802154_HW_FILTER not available) */
+	if (!(radio->get_capabilities(net_if_get_device(iface)) & IEEE802154_HW_FILTER)
+			&& !ieeee802154_check_dst_addr(iface, &mpdu.mhr)) {
```

Comment:
nit: Please align to the opening bracket
```suggestion
	if (!(radio->get_capabilities(net_if_get_device(iface)) & IEEE802154_HW_FILTER) &&
	    !ieeee802154_check_dst_addr(iface, &mpdu.mhr)) {
```